contracts/SuiBridge.sol

```diff
@@ -25,7 +26,15 @@ contract SuiBridge is ISuiBridge, CommitteeUpgradeable, PausableUpgradeable {
     IWETH9 public weth9;
     // message nonce => processed
     mapping(uint64 => bool) public messageProcessed;
-
+    mapping(uint8 chainId => bool isSupported) public supportedChainIDs;
+    
+    modifier isTargetChainSupported(uint8 targetChainID) {
```

Comment:
Because we're using this modifier to check target chain and destination chains, lets just call it `isChainSupported`